src/viam/examples/modules/complex/gizmo/api.cpp

```diff
@@ -22,8 +22,8 @@ GizmoRegistration::GizmoRegistration(const google::protobuf::ServiceDescriptor*
     : ResourceRegistration(service_descriptor){};
 
 std::shared_ptr<ResourceServer> GizmoRegistration::create_resource_server(
-    std::shared_ptr<ResourceManager> manager) {
-    return std::make_shared<GizmoServer>(manager);
+    std::shared_ptr<ResourceManager> manager, std::shared_ptr<Server> server) {
+    return std::make_shared<GizmoServer>(manager, server);
 };
```

Comment:
As mentioned in the PR description, general pattern here for all `create_resource_server` methods is to accept a `std::shared_ptr<Server>` and pass it to construction of the `[TypeOfResource]Server`.